.travis.yml

```diff
@@ -1 +1,3 @@
 language: java
+jdk: oraclejdk8
+dist: trusty
```

Comment:
both new lines might be removed after the transition to java 11 (default on Travis)